flink-metrics/flink-metrics-prometheus/src/main/java/org/apache/flink/metrics/prometheus/AbstractPrometheusReporter.java

```diff
@@ -378,6 +390,22 @@ private void addSamples(
                                 addToList(labelValues, quantile.toString()),
                                 statistics.getQuantile(quantile)));
             }
+            if (this.histogramMaxEnabled) {
+                samples.add(
+                        new MetricFamilySamples.Sample(
+                                metricName,
+                                labelNamesWithQuantile,
+                                addToList(labelValues, "1.0"),
+                                statistics.getMax()));
+            }
+            if (this.histogramMinEnabled) {
+                samples.add(
+                        new MetricFamilySamples.Sample(
+                                metricName,
+                                labelNamesWithQuantile,
+                                addToList(labelValues, "0.0"),
+                                statistics.getMin()));
+            }
```

Comment:
Core of the changes, emit max as 1.0 and min as 0.0 when reporting to prometheus